frontends/ast/simplify.cc

```diff
@@ -780,6 +782,100 @@ bool AstNode::simplify(bool const_fold, bool at_zero, bool in_lvalue, int stage,
 		delete_children();
 	}
 
+	// resolve typedefs
+	if (type == AST_TYPEDEF) {
+		log_assert(children.size() == 1);
+		log_assert(children[0]->type == AST_WIRE || children[0]->type == AST_MEMORY);
+		while(children[0]->simplify(const_fold, at_zero, in_lvalue, stage, width_hint, sign_hint, in_param)) {
+			did_something = true;
+		};
```

Comment:
Style nitpick: No `{..}` for a single statement and certainly no `};`.  ;)

Reply:
Oops, will fix once next at a PC